### DISCUSSION TITLE
Difficult Scrolling and Animations - Any Driver Alternavives?

### DISCUSSION THREAD

**Discussion by trey-a-12** [utmapp/UTM#5145] [UTM Help (macOS)]
**TLDR Version: Scrolling is jittery on trackpad (in UTM and worse in VMware while great in Parallels), and animations are sort of glitchy while fast (in UTM, great on Parallels AND somehow VMware). Any trackpad and/or display drivers or fixes? Also, the resolution keeps resetting for some reason.**

When looking into running Windows on my M1 MBA, I frequently saw UTM articles and videos, eventually deciding it was the best choice for me. After eventually trying out Parallels and VMware Fusion 13 however, I'm stuck. See, obviously Parallels is the best due to 3D Acceleration, but that doesn't explain why scrolling is so natural in it, while UTM scrolling jitters on both my built-in and Apple Magic Trackpad 1 _(though still better than VMware's unusable trackpad experience lol)_.

I get that there's no trackpad shown in display preferences, but there doesn't appear to be one in Parallels or VMware either, so has anyone found an alternative driver? I looked at the https://github.com/imbushuo/mac-precision-touchpad driver, which did wonders compared to the Boot Camp drivers on an old Intel Mac, but installing them here says "Found and installed on 0 devices" or something. No matter what I try, the trackpad is a horrible experience when scrolling here.

The other issue is animations. Again, of course Parallels excels due to 3D acceleration so everything looks as good as on a PC, but somehow, even VMware's animations actually look smooth most of the time, while here I'm just left thinking "at least the trackpad's usable", otherwise I'd be a VMware user for Windows on Apple Silicon.

Before you ask, yes, I've installed the Windows Guest Tools and normal Spice Guest Tools both, and everything works - it just looks to be of subpar quality. **_Has anyone found any alternative drivers or applications to possibly help the trackpad or animations?_**

Oh, and every time I boot, UTM thinks I want to set my resolution to 1440x900, though I specifically change it to 1920x1200 or other resolutions every time - **Even though I also change the display scale every time and it keeps that.** I think that has something to do with the fact that it detects 0 MB of video memory though, and how the active signal mode is 1024x768. Fonts are NOT smooth in most cases, another issue unique to UTM.

**Device Specs:**

- M1 MacBook Air
- 8GB RAM (4 dedicated to VM)
